src/Support/MarkdownParser.php

```diff
@@ -49,7 +49,7 @@ private function stripUnalllowedTags(string $html): string
     {
         return strip_tags(
             $html,
-            ['a', 'p', 'br', 'ul', 'ol', 'li', 'strong', 'em', 'svg', 'g', 'path'],
+            ['a', 'p', 'br', 'ul', 'ol', 'li', 'strong', 'em', 'svg', 'g', 'path', 'div'],
```

Comment:
no `div` or `span` as that will cause issues with the layout as it will allow people to close arbitrary divs when it ends up in the DOM

Reply:
yes already looking for alternatives